roundabout/globals.js

```diff
@@ -0,0 +1,8 @@
+// NOTE: shim globals needed by content-claims client deps that would be present in nodejs v18.
+// TODO: migrate to sst v2 and nodejs v18+
+import { TransformStream, WritableStream, CountQueuingStrategy } from 'node:stream/web'
+import { fetch } from 'undici'
+globalThis.TransformStream = TransformStream
+globalThis.CountQueuingStrategy = CountQueuingStrategy
+globalThis.WritableStream = WritableStream
+globalThis.fetch = fetch
```

Comment:
migrating to sst v2 is not trivial, so this seems to be the least worst workaround I can think of for now.

Reply:
Mentioned in https://github.com/web3-storage/w3infra/pull/152 to guarantee we do not forget to take these